x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/list_page/index.tsx

```diff
@@ -70,61 +162,116 @@ export const AgentConfigListPage: React.FunctionComponent<{}> = () => {
+          <EuiFlexGroup gutterSize="s" wrap={true} alignItems="baseline">
+            <EuiFlexItem grow={false}>
+              <EuiLink href={`${DETAILS_URI}${agentConfig.id}`}>{name || agentConfig.id}</EuiLink>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText color="subdued" size="xs">
                 <FormattedMessage
-                  id="xpack.ingestManager.agentConfigList.viewActionLinkText"
-                  defaultMessage="view"
+                  id="xpack.ingestManager.agentConfigList.revisionNumber"
+                  defaultMessage="rev. {revNumber}"
+                  values={{ revNumber: '999' }}
                 />
-              </EuiLink>
-            );
-          },
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        ),
+      },
+      {
+        field: 'description',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.descriptionColumnTitle', {
+          defaultMessage: 'Description',
+        }),
+        render: (description: AgentConfig['description']) => (
+          <EuiTextColor color="subdued">{description}</EuiTextColor>
+        ),
+        width: '30%',
+      },
+      {
+        field: 'updated_on',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.updatedOnColumnTitle', {
+          defaultMessage: 'Last updated on',
+        }),
+        render: (date: AgentConfig['updated_on']) => (
+          <FormattedDate value={date} year="numeric" month="long" day="2-digit" />
+        ),
+      },
+      {
+        field: 'agents',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.agentsColumnTitle', {
+          defaultMessage: 'Agents',
+        }),
+        render: (agents: unknown, config: AgentConfig) => {
+          // FIXME: implement agents once known in API/Schema
+          const agentCount = 99;
+          const displayValue = (
+            <FormattedMessage
+              id="xpack.ingestManager.agentConfigList.agentsText"
+              defaultMessage="{agentCount, plural, one {# agent} other {# agents}}"
+              values={{ agentCount }}
+            />
+          );
+          return agentCount > 0 ? (
+            <EuiLink href={`${FLEET_URI}?kuery=agents.config_id : ${config.id}`}>
+              {displayValue}
+            </EuiLink>
+          ) : (
+            displayValue
+          );
         },
-      ],
-      width: '100px',
-    },
-  ];
+      },
+      {
+        field: 'datasources',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.datasourcesCountColumnTitle', {
+          defaultMessage: 'Data sources',
+        }),
+        render: (datasources: AgentConfig['datasources']) => (datasources ? datasources.length : 0),
+      },
+      {
+        name: i18n.translate('xpack.ingestManager.agentConfigList.actionsColumnTitle', {
+          defaultMessage: 'Actions',
+        }),
+        width: '100px',
```

Comment:
remove width property so that all columns (except for description) are distributed evenly

Reply:
Actually, can you also set the "name" column to be `width: '20%'` as well

Reply:
both were done 😃  and as discussed via Slack - I also added text truncate to the Agent Config Name